frontend/packages/client/package.json

```diff
@@ -3,14 +3,15 @@
   "version": "1.0.1",
   "private": true,
   "dependencies": {
+    "@craco/craco": "^6.4.5",
     "@creativebulma/bulma-tooltip": "^1.2.0",
     "@hookform/resolvers": "^2.9.7",
     "@hotjar/browser": "^1.0.6",
     "@onflow/fcl": "^1.2.0",
     "@onflow/sdk": "^1.1.1",
     "@onflow/six-transfer-tokens": "0.0.8",
-    "@tanstack/react-query": "^4.1.0",
-    "@tanstack/react-query-devtools": "4.0.11-beta.0",
+    "@tanstack/react-query": "4.3.3",
+    "@tanstack/react-query-devtools": "4.3.3",
```

Comment:
had to specify exact versions because I was running into this bug that was seemingly introduced yesterday: https://github.com/TanStack/query/issues/4155